platform/mock-radio.ts

```diff
@@ -58,16 +58,43 @@ export class MockNetwork {
   }
 }
 
-export class UDPDevice implements MockUDPListener {
-  private udpMessageAction: (msg: Buffer, rinfo: RemoteAddressInfo) => void;
+export class UDPDevice {
+  private deviceId: string;
+  private port: number;
+  private address: string;
+  private network: MockNetwork;
 
-  onUDPMessage(msg: Buffer, rinfo: RemoteAddressInfo): void {
-    this.udpMessageAction(msg, rinfo);
+  public constructor(
+    deviceId: string,
+    network: MockNetwork,
+    port: number,
+    address: string
+  ) {
+    this.deviceId = deviceId;
+    this.network = network;
+    this.port = port;
+    this.address = address;
   }
 
-  setUDPMessageAction(
-    messageAction: (msg: Buffer, rinfo: RemoteAddressInfo) => void
-  ) {
-    this.udpMessageAction = messageAction;
+  public getDeviceId(): string {
+    return this.deviceId;
+  }
+
+  public getDevicePort(): number {
+    return this.port;
+  }
+
+  public triggerIdentify(
```

Comment:
This triggers an IDENTIFY intent? Could be helpful to include some JSDoc.

Reply:
Good point this isn't directly clear.  I renamed it to `sendDiscoveryBuffer` since that's more accurate.  The `MockLocaIHomePlatform` is what receives the `discoveryPlatform` and triggers the IDENTIFY request.  Also added a JSDoc.  I'm planning on adding JSDocs later, but I'll start adding JSDocs here and there as to not do it all at once (see #20).